server/channels/db/migrations/postgres/000110_create_desktop_tokens.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS desktoptokens (
+    token VARCHAR(64) NOT NULL,
+    createat BIGINT NOT NULL,
+    userid VARCHAR(26) NOT NULL,    
+    PRIMARY KEY (token)
+);
+
+CREATE INDEX IF NOT EXISTS idx_desktoptokens_token_createat ON desktoptokens(token, createat)
```

Comment:
```suggestion
CREATE INDEX CONCURRENTLY IF NOT EXISTS idx_desktoptokens_token_createat ON desktoptokens(token, createat)
```

Reply:
Getting this error with `CONCURRENTLY`:

`Error: failed to initialize platform: cannot create store: failed to apply database migrations: driver: postgres, message: failed to execute migration, command: executing_query, originalError: pq: CREATE INDEX CONCURRENTLY cannot run inside a transaction block`

Reply:
Oops yeah my bad. You need to move this statement to a separate file and add a `// morph:nontransactional` comment on top. But since this is just creating an index on a new table, it should be fine to skip it.